validator/client/validator.go

```diff
@@ -938,7 +938,7 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 	}
 	for i := types.Slot(0); i < params.BeaconConfig().SlotsPerEpoch; i++ {
 		startTime := slots.StartTime(v.genesisTime, slotOffset+i)
-		durationTillDuty := time.Until(startTime)
+		durationTillDuty := time.Until(startTime) + time.Second
```

Comment:
always afraid of these kinds of changes that touch time...
startTime is a time object which includes milliseconds and nano seconds , what display rounds down this to 11 seconds?
it'll be a little less than 12 seconds but probably isn't a full second off
time from slot to slot - some time on the function. 

if we add a full second here it'll be a bit more than 12 seconds, is that ok?

Reply:
To clarify, this is only cosmetic, if you look down: `"timeTillDuty":          durationTillDuty.Round(time.Second),`

It wont be more than 12 seconds, it's always one sec less